distribution/openrct2.d.ts

```diff
@@ -2076,6 +2076,54 @@ declare global {
          * The min chain lift speed for this ride in miles per hour.
          */
         readonly minLiftHillSpeed: number;
+
+        /**
+         * The max speed. (speed * 9) >> 18 = miles per hour.
+         */
+        readonly maxSpeed: number;
```

Comment:
Wouldn't it be nicer to return the speed in miles per hour here? I guess it wouldn't be as accurate. Perhaps a custom `Speed` type would be even better.

Reply:
There's a lot of these unreadable values that are coming in, each with their own formulas, and some are in metric while others are imperial. From a plugin dev pov I think it would be easier if I just converted everything to metric here.

Reply:
@Sadret initially wanted them returned in their original form and letting the plugin devs format them. Thoughts?

Reply:
In my opinion, the best solution is to have all stats scaled such that they can be used within the `formatString` method, in this case with `"{VELOCITY}"`.